source/SIL.AppBuilder.Portal.Frontend/src/ui/components/inputs/date-range/styles.scss

```diff
@@ -0,0 +1,29 @@
+.dateRange {
+  position: relative;
+  font-size: 11px;
+  color: #0c3855;
+  font-weight: bold;
+  opacity: 0.8;
+
+}
+
+
+.input {
```

Comment:
this is going to affect every single input that had a svg on it. Is this are standard? if so we should move this to app.scss

Reply:
semantically, this should only affect the date-range input/svg. 
If that's not the case, we have a problem with how our styles are bundled